templates/case/decision/index.html.twig

```diff
@@ -9,13 +9,14 @@
         <div class="col">
             <h1>{% trans %}Decision{% endtrans %}</h1>
         </div>
-        <div class="col-auto">
-            {% if hasNoActiveParty %}
-                <button class="btn btn-success" disabled title="{% trans %}Cannot create decision since case has no party to send it to{% endtrans %}">{% trans %}Create decision{% endtrans %}</button>
-            {% else %}
-                <a class="btn btn-success" href="{{ path('case_decision_create', {'id': case.id}) }}" role="button">{% trans %}Create decision{% endtrans %}</a>
-            {% endif %}
-        </div>
+        {# TODO: Reintroduce at a later stage #}
+{#        <div class="col-auto">#}
+{#            {% if hasNoActiveParty %}#}
+{#                <button class="btn btn-success" disabled title="{% trans %}Cannot create decision since case has no party to send it to{% endtrans %}">{% trans %}Create decision{% endtrans %}</button>#}
+{#            {% else %}#}
+{#                <a class="btn btn-success" href="{{ path('case_decision_create', {'id': case.id}) }}" role="button">{% trans %}Create decision{% endtrans %}</a>#}
+{#            {% endif %}#}
+{#        </div>#}
```

Comment:
Consider using just a single `{#` `#}` pair to make reintroducing later easier: 

```twig
{#
…
#}
```

Reply:
Updated.